client/app/views/popover_screens/guests.coffee

```diff
@@ -70,6 +70,15 @@ module.exports = class GuestPopoverScreen extends EventPopoverScreenView
             @$('input[name="guest-name"]').focus()
 
 
+    # Link directly click action to related guest, avoid using Backbone's
+    # default event managing, as we are here in a sort of CollectionView.
+    bindGuestActions: ($element, guest) ->
+        $element
```

Comment:
If you dont use Backbone's event managing, you need to manually unbind in onRemoveGuest as well. 

Why not use list-level listener and data-guestid attributes intead ?